lib/Backends/CPU/LLVMIRGen.h

```diff
@@ -140,8 +149,8 @@ class LLVMIRGen {
   /// the type specified by \p kind.
   llvm::Value *emitConst(llvm::IRBuilder<> &builder, float val,
                          glow::ElemKind kind);
-  /// Generates LLVM IR that materializes the constant array \p vals. Note that
-  /// it will cast non-size_t types T into size_t.
+  /// Generates LLVM IR that materializes the constant array \p vals. Note
+  /// that it will cast non-size_t types T into size_t.
```

Comment:
All the formatting changes in comments here and below seem to be unrelated to your changes. Please remove them.

Reply:
Clang format did this to me... I'll split to a different commit maybe.